components/items/ItemsGrid/useItemsGrid.ts

```diff
@@ -232,6 +235,46 @@ export function useFetchSearch({
     resetSearch()
   })
 
+  const processOnchainData = useDebounceFn(async () => {
+    items.value = await Promise.all(items.value.map(async (item) => {
+      if ('onchainData' in item && item.onchainData) {
```

Comment:
```suggestion
      if (item.onchainData) {
```
It should be enough.

Reply:
need additional `'onchainData' in item` is because that custom key is doesn't exists in the types. I need to add to the types with `item.onchainData`. lmk which one do you prefer